src/android/CodePush.java

```diff
@@ -46,32 +46,36 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
 
     @Override
     public boolean execute(String action, CordovaArgs args, final CallbackContext callbackContext) {
-        if ("getServerURL".equals(action)) {
-            this.returnStringPreference("codepushserverurl", callbackContext);
-            return true;
+        if ("getAppVersion".equals(action)) {
```

Comment:
actually, can we change this awful if/else chain into a switch statement? thanks for alphabetizing btw